app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -19,7 +19,8 @@ package com.duckduckgo.app.pixels
 import com.duckduckgo.app.statistics.pixels.Pixel
 
 enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
-    APP_LAUNCH("ml"),
+    APP_LAUNCH_LEGACY("ml"),
+    APP_LAUNCH("m_app_launch_c"),
```

Comment:
Should this have a more descriptive name? Not sure what the "c" is for.

Reply:
we've been using "c" for the count pixel, but I don't think we need it here.